test/tests.zig

```diff
@@ -1567,10 +1567,6 @@ pub fn addCAbiTests(b: *std.Build, options: CAbiTestOptions) *Step {
             });
             for (c_abi_target.c_defines) |define| test_step.defineCMacro(define, null);
 
-            // This test is intentionally trying to check if the external ABI is
-            // done properly. LTO would be a hindrance to this.
-            test_step.want_lto = false;
```

Comment:
As long as `want_lto` exists, this should be kept, since the default value is irrelevant - this test needs it to be off independent of the default value.